charts/admission-controller/values.yaml

```diff
@@ -2,18 +2,31 @@
 # This is a YAML-formatted file.
 # Declare variables to be passed into your templates.
 
+global:
+  # Global cluster config options
+  clusterConfig: {}
+  sysdig:
+    # Global API token to access Sysdig Secure.
+    secureAPIToken: ""
+    # Global secret with API Token to access Sysdig Secure.
+    secureAPITokenSecret: ""
+  # Global HTTP Proxy settings.
+  proxy: {}
 
 # **required**
 # <br/>Cluster Name which appear on Secure UI
 clusterName: ""
 
-sysdig:
+# Namespace to install components, if not specified `Release.Namespace` is used.
+namespace: ""
```

Comment:
I did a test by setting a different namespace for the admission controller and in case the namespace doesn't already exist the whole helm installation fails:
```
Error: INSTALLATION FAILED: namespaces "test" not found
```
this is the command used:
```
 ➜ helm install sysdig-agent -f values-test.yaml charts/sysdig-deploy -n sysdig-agent --create-namespace
```

values-test.yaml
```
global:
  clusterConfig:
    name: "test-cluster"
  sysdig:
    accessKeySecret: "sysdig-agent"
    region: "us1"
  image:
    registry: quay.io
  proxy:
    httpProxy: http://squid.mydomain.tld:3128
    httpsProxy: http://squid.mydomain.tld:3128
    noProxy: 100.96.0.0/10

admissionController:
  enabled: true
  namespace: "test"
  sysdig:
    secureAPIToken: test

nodeAnalyzer:
  nodeAnalyzer:
    benchmarkRunner:
      deploy: false
      resources:
        limits:
          cpu: 250m
    runtimeScanner:
      deploy: true
      eveConnector:
        deploy: true
      resources:
        limits:
          cpu: 300m
      settings:
        eveEnabled: true
  secure:
    vulnerabilityManagement:
      newEngineOnly: true
```

I don't like too much the idea of giving the possibility to specify a different namespace while using sysdig-deploy chart but seems to be fine for the other maintainers so we can keep it :) 
Just please ensure to add a note on the `readme.md` file that for the specified namespace, it must exist prior to apply/install the chart

Reply:
good point. this is a problem. may need to discuss with the product team before committing to this.

Reply:
i have added a note for the namespace flag, it should be quite clear now